CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 - Implement `Default`, `Hash`, and `Eq` for `LogicalPosition`, `PhysicalPosition`, `LogicalSize`, and `PhysicalSize`.
 - On macOS, initialize the Menu Bar with minimal defaults. (Can be prevented using `enable_default_menu_creation`)
 - On macOS, change the default behavior for first click when the window was unfocused. Now the window becomes focused and then emits a `MouseInput` event on a "first mouse click".
+- Implement mint (math interoperability standard types) conversions.
```

Comment:
Please add a note that it's behind a feature gate.
Also, an entry in the README would be nice (even if it's not up-to-date): https://github.com/rust-windowing/winit#cargo-features